modules/special-rules/pages/tyranid-objectives.adoc

```diff
@@ -0,0 +1,16 @@
+= Tyranid Objectives
+
+Always ready for a krumpin, Orks will often charge into melee before the enemy has even realised that the Orks are close enough to charge them!
+
+---
+
+Apply these rules to Tyranid armies:
```

Comment:
```suggestion
When you field a Tyranid army, apply these rules:
```
The construction in this suggestion supports other changes below.

Reply:
See A Dying Race. I don't think it's a good idea to arbitrarily change how the abilities are constructed.

However I thought this was a specific enough case that we didn't need to use the full 'Apply these rules to [units/detachments/armies] with the {keyword-attribute} ability:' version, and save ourselves a {keyword-attribute} that's going to be very rarely used.

Reply:
Suggestion added to commit batch.